packages/react-tabs/Spec.md

```diff
@@ -141,65 +138,84 @@ tabs are stacked vertically, tab content is stacked horizontally
 </TabList>
 ```
 
-### App Bar
+### Vertical Tab content
 
-Tab content is stacked vertically
-
-> TODO: Verify with design that tabs in appbar layout are meant to integrate into some vertical command bar later?
-> TODO: Ask design about app bar and difference between allowing tabs to layout vertically
-> TODO: Ask design if appbar can ever be horizontal layout
-> TODO: Ask design if transparent | subtle appearance can be applied to appBar rather than appBar having distinct appearance
+- tab content is arranged vertically
 
 ```tsx
-<TabList appBar>
+<TabList verticalContent>
   <Tab>One</Tab>
   <Tab>Two</Tab>
   <Tab>Three</Tab>
 </TabList>
-```
-
--or-
 
-```tsx
-<TabList vertical>
-  <Tab vertical>One</Tab>
-  <Tab vertical>Two</Tab>
-  <Tab vertical>Three</Tab>
+<TabList vertical verticalContent>
+  <Tab>One</Tab>
+  <Tab>Two</Tab>
+  <Tab>Three</Tab>
 </TabList>
 ```
 
 ### Appearance
 
-> TODO: Confirm with design if transparent is the default.
-> TODO: Confirm with design there is not a manilla folder tab appearance.
-> TODO: Ask design what the subtle and transparent styles are for focus, hover, pressed, disabled.
+Transparent (default)
+
+- no border and background styles
+- selection is indicated by a primary line below the tab
+- hovering over a tab shows a secondary line below the tab
 
 ```tsx
-<TabList>{/* ... */}</TabList>
+<TabList>
+  <Tab />
+  <Tab />
+  <Tab />
+</TabList>
 
 <TabList appearance="transparent">{/* ... */}</TabList>
 ```
 
+Subtle
+
+- similar to transparent, but a light background is set when hovering over a tab
+
 ```tsx
 <TabList appearance="subtle">{/* ... */}</TabList>
 ```
 
-### Badges
+### With Icon
 
-> TODO: How to support multiple badge types as a slot?
-> TODO: How to specify icon for basic badge in a slot?
+- icon is positioned before the tab content
+
+```tsx
+<TabList>
+  <Tab icon={<CheckboxComposite />}>Allowed</Tab>
+  <Tab icon={<BlockedSite />}>Blocked</Tab>
+</TabList>
+```
+
+- icon only when content omitted
+
+```tsx
+<TabList>
+  <Tab icon={<CheckboxComposite />} />
+  <Tab icon={<BlockedSite />} />
+</TabList>
+```
+
+### With Badge
 
 ```tsx
 <TabList>
   <Tab>Files</Tab>
   <Tab>Search</Tab>
-  <Tab badge={<CounterBadge count={5} />}>Messages</Tab>
+  <Tab badge={<Badge>New</Badge>}>Semantic Search</Tab>
 </TabList>
 ```
 
 ### Overflow
 
-Handling overflow is a built-in feature of tabs and replaceable via a slot.
+- overflow is a menu button
+- replaceable via a slot
 
 ```tsx
 <TabList overflow={</MyOverflowButton}>
```

Comment:
How can the menu be rendered and customized with the `overflow` slot ? is the `MyOverflowButton` component simply a button that an internal `Menu` in `TabList` uses as a trigger ? or is `MyOverflowButton` the entire `Menu` component with a trigger ?

Reply:
I just saw that the `overflowButton` slot is a shorthand of `MenuProps`. That _**could**_ work. However it leaves out the opportunity to configure any of the children of `Menu`. The default usage of `Menu` is something like this:

```tsx
<Menu>
  <MenuTrigger>
    <Button />
  </MenuTrigger>
  
  <MenuPopover>
    <MenuList>
      <MenuItem />
      <MenuItem />
      <MenuItem />
      <MenuItem />
    </MenuList>
  </MenuPopover>
</Menu>
```